tests/integration/test_ethereum_tester.py

```diff
@@ -151,6 +168,21 @@ def unlocked_account(web3, unlockable_account, unlockable_account_pw):
     web3.personal.lockAccount(unlockable_account)
 
 
+@pytest.fixture(params=['bytes', 'hex'])
+def unlockable_account_dual_type(unlockable_account, request):
+    if request.param == 'bytes':
+        return to_bytes(hexstr=unlockable_account)
+    else:
+        return unlockable_account
+
+
+@pytest.fixture
+def unlocked_account_dual_type(web3, unlockable_account_dual_type, unlockable_account_pw):
```

Comment:
I liked the other approach in this PR: replace the existing fixture with a parametrized one, as opposed to leaving the old one and adding a new one with a different name.

Reply:
I had to do this because some tests were failing with the parametrized approach. Its the same issue that I've mentioned in https://github.com/ethereum/web3.py/pull/893#issuecomment-396145172.